src/handlers/playbookDispatcher/index.ts

```diff
@@ -37,10 +37,9 @@ function parseMessage(message: Message) {
 export default async function onMessage(message: Message) {
     const eventType = message.headers?.event_type?.toString();
 
-    // Only handle 'update' events; ignore 'create', 'delete', and 'read' events
-    // 'create' is handled separately when playbook run is initially created because the status is always 'running'
+    // Only handle 'update' and 'create' events; ignore delete', and 'read' events
     // 'delete' and 'read' are irrelevant for status updates
-    if (eventType === 'update') {
+    if (eventType === 'create' || eventType === 'update') {
```

Comment:
This is assuming that the playbook run was already created in our DB so we're not actually creating the playbook_runs entry here, we're just updating the status that we received in the Kafka event